src/features/movieDetails/components/MovieDetailsWrapper/index.js

```diff
@@ -41,11 +59,25 @@ export const MovieDetailsWrapper = ({
                     <Title>{title}</Title>
                     <Year>{yearShort}</Year>
                     <NameBlock>
-                        <GreyName>Production:</GreyName>
-                        <DescriptionName>
-                            {production}
-                        </DescriptionName>
-                        <GreyName>Release date:</GreyName>
+                        {windowWidth >= 390 ? (
```

Comment:
Instead of checking windowWidth >= 390 multiple times in your JSX, you can compute the appropriate values beforehand and then use them directly in your JSX:
```
const isWideScreen = windowWidth >= 390;
    const showProduction = isWideScreen ? production : shortProduction;
    const showReleaseDate = isWideScreen ? yearFull : null;
```